pkgs/applications/networking/diswall/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "diswall";
+  version = "0.2.0";
```

Comment:
https://github.com/dis-works/diswall-rs/commit/0509bf696c5442ccf91cddaeaeb70de9560c85df#diff-94da6253c6e14901145d97aaaa22be83023353976134f4ef9293bf4eb5e257c7R116
Would this even work on NixOS? `/etc/systemd/system` is a symlink to `/nix/store` which is read only.

Reply:
Haven't tested yet. I haven't get there to creating a diswall service yet.